sdk/cognitiveservices/cognitiveservices-qnamaker/README.md

```diff
@@ -18,21 +18,30 @@ npm install @azure/cognitiveservices-qnamaker
 #### nodejs - Authentication, client creation and getKeys endpointKeys as an example written in TypeScript.
 
 ##### Sample code
+The following sample lists the keys of the QnAMaker endpoint. To know more, refer to the [Azure Documentation on QnAMaker](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/)
 
 ```typescript
 import * as msRest from "@azure/ms-rest-js";
-import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
-import { QnAMakerClient, QnAMakerModels, QnAMakerMappers } from "@azure/cognitiveservices-qnamaker";
-const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
-const endpoint = "https://westus.api.cognitive.microsoft.com"; // OR some other endpoint.
-const creds = new msRest.ApiKeyCredentials({ inHeader: { 'Ocp-Apim-Subscription-Key': subscriptionId }});
-const client = new QnAMakerClient(creds, endpoint);
-client.endpointKeys.getKeys().then((result) => {
-  console.log("The result is:");
-  console.log(result);
-}).catch((err) => {
-  console.error(err);
+import { QnAMakerClient } from "@azure/cognitiveservices-qnamaker";
+
+const qnaMakerKey = process.env["YOUR_QNAMAKER_KEY"];
+const endpoint =
+  process.env["YOUR_QNAMAKER_ENDPOINT"] || "<YOUR_QNAMAKER_ENDPOINT>";
+
+const creds = new msRest.ApiKeyCredentials({
```

Comment:
Earlier offline suggestion to use `ApiKeyCredentials` instead of `CognitiveServiceCredentials` was assuming that we will not be re-generating the sdk and updating the version of ms-rest-js. Now that we are indeed doing so, we should use `CognitiveServiceCredentials` here to be in sync with the samples for all the other Cognitive Services libraries

Reply:
Done